flexget/plugins/input/filmweb_watchlist.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import unicode_literals, division, absolute_import
+
+import logging
+
+from flexget import plugin
+from flexget.event import event
+from flexget.utils.cached_input import cached
+from flexget.entry import Entry
+
+try:
+    from filmweb.filmweb import Filmweb as FilmwebAPI
+    from filmweb.items import LoggedUser
+    from filmweb.exceptions import RequestFailed
+except ImportError:
+    pass
+else:
```

Comment:
This pattern is kinda weird. We have a `DependencyError` exception for scenarios like this one